inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
Its not about optimizing this case, its about setting a standard practice to follow, so that any policy can be easily toggled, and you can know what to expect because they are all enabled or disabled the same way.